cfgov/v1/atomic_elements/organisms.py

```diff
@@ -6,6 +6,13 @@
 from ..util import ref
 from ..models.snippets import Contact as ContactSnippetClass
 
+JS_ORGANISMS = [
```

Comment:
I'm not sure, but this is changing in a follow PR where I'm using Wagtail's hooks module instead of importing the handlers and adding them to a list. Much cleaner.